### ISSUE TITLE
A need for death to change

### ISSUE THREAD

**Issue by SteamDoge9000** [GTNewHorizons/GT-New-Horizons-Modpack#10587]
### Your GTNH Discord Username

Shibva_

### Your Pack Version

doe sit even matter, its the one with the logitstic thing causuing a false possative ok? 

### Your Proposal

Lets be honnest, the grave system added by OpenBlcoks is horrid; many possibly can agree with this and it may be time for something to change about it

when we think of gravestone mods we think something among the lines of mods like Corail Tombstone, that allows for players to go back to their grave or even teleport to it at a cost of a ender pearl.
whether this is balanced or not is up for the nerf neards but the fact is that its annoying to deal with the graves in this pack that many prefer to set gamerule keepinventory = true takes some of the difficulty on it

IM saying this prom personal experience, when KeepINventory is set to true, not only does it save your stuff, it also acts a a "poor mans" teleport back to their respawn point

take this in whatever way you want to think about it, but I think most of us that have played this pack can agree that the graves could use a overhaul

whether its to backport or dramatically rework the one in OpenBlcoks is up to the developers.

### Your Goal

Better graves to allow for there to sill be a use for playing with keepinventory false to allow for some punishment to still happen or just punishment for death in general

### Your Vision

the top should explain it; I forgot about this part 

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by SteamDoge9000** [comment]
One  👍 = 1 person that agrees

**Comment by YannickMG** [comment]
I don't know, in the early game having all my important stuff stuck in a roguelike dungeon right by a creeper spawner made for quite the adventure getting it back.

Figuring out how to extract the grave you left in a dangerous situation can be it's own fun logistical problem.

**Comment by boubou19** [comment]
Current grave system is ok, just the handling of the graves for admins suck. That's why we use amdiforge on the officials.

Honestly going in a rogue like dungeon without being prepared is suicidal. So is the exploitation of any spawner early game.

**Comment by YannickMG** [comment]
> Honestly going in a rogue like dungeon without being prepared is suicidal.

Right? That's what made it fun. In my defense I thought I was prepared.

**Comment by SteamDoge9000** [comment]
ive use Corlail Tombstone and it can actually make you invisible for a time to all mobs

but there's a catch, the moment you do anything like open a chest or attack a mob or even open your own gravel the effect is removed "

I was doing a non-legit play with a hellpack I made involving zombie awareness and scape and run parasites and scaling health

while you can teleport back to your grave, the entities may still linger 

in short, tping back may result in you dying again

**Comment by boubou19** [comment]
are you really asking for a mod addition into the pack without following the template for it?

**Comment by SteamDoge9000** [comment]
huh? oh, whopse guess I used the wrong template
